bot/seasons/valentines/movie_generator.py

```diff
@@ -0,0 +1,61 @@
+import discord
+import logging
+from discord.ext import commands
+import random
+import json
+from urllib import parse, request
+from os import environ
+log = logging.getLogger(__name__)
+
+
+class MovieGenerator:
+    """
+    a cog that gives random romance movie suggestion to a user
+
+    """
+
+    def __init__(self,bot):
+        self.bot = bot
+
+    @commands.command()
+    async def movie(self, ctx):
```

Comment:
Need a docstring so there is help for this command

Reply:
I made changes can you review it please ?